pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -650,16 +646,14 @@ func (computeCfg *computeServiceConfig) getATGroupView(nepheControllerATSGNameTo
 				EgressRules:    nepheControllerATSgNameToEgressRulesMap[atSgName],
 			}
 			enforcedSecurityCloudView = append(enforcedSecurityCloudView, groupSyncObj)
-
-			nepheControllerATSgNameSet[atSgName] = struct{}{}
 		}
 	}
 
-	return enforcedSecurityCloudView, nepheControllerATSgNameSet, nil
+	return enforcedSecurityCloudView, nil
 }
 
 func (computeCfg *computeServiceConfig) getAGGroupView(nepheControllerAGSgNameToCloudResourcesMap map[string][]securitygroup.CloudResource,
```

Comment:
nit: please add comment as and when you touch some code which doesnt have it.

Reply:
added.